### PR TITLE
Adds WPScan CVEs

### PR DESCRIPTION
**Pull request by erwanlr**

### CODE REVIEW

File: 2022/1xxx/CVE-2022-1091.json [hunk 1]
```diff
@@ -1,18 +1,80 @@
 {
```
Comment (cve-team) [inline 2022/1xxx/CVE-2022-1091.json:1]: Invalid source.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/1xxx/CVE-2022-1091.json (hunk 1 above)
Comment (erwanlr) [reply 2022/1xxx/CVE-2022-1091.json:1]: I don't see what the issue is on that one oO

File: 2022/1xxx/CVE-2022-1091.json [hunk 2]
```diff
@@ -1,18 +1,80 @@
...
+  },
+  "references": {
+    "reference_data": [
+      {
+        "refsource": "MISC",
+        "url": "https://wpscan.com/vulnerability/4d12533e-bdb7-411f-bcdf-4c5046db13f3",
+        "name": "https://wpscan.com/vulnerability/4d12533e-bdb7-411f-bcdf-4c5046db13f3"
+      },
+      {
+        "refsource": "CONFIRMED",
```
Comment (anthnysingleton) [inline 2022/1xxx/CVE-2022-1091.json]: This should be "CONFIRM" and not "CONFIRMED"

Comment (erwanlr) [reply 2022/1xxx/CVE-2022-1091.json]: Haaa it's that source, I was looking at

```
"source": {
    "discovery": "EXTERNAL"
  }
```

Thank you very much!!

Will update that in a sec

**Pull request merged**